sympy/functions/special/delta_functions.py

```diff
@@ -99,16 +101,27 @@ def simplify(self, x):
         """
         from sympy.polys.polyroots import roots
 
-        if not self.args[0].has(x) or (len(self.args) > 1 and self.args[1] != 0 ):
+        wrt = hints.get('wrt', None)
+        if wrt is None:
```

Comment:
I think here we should add : 
```py
wrt = hints.get('wrt', False)
if wrt is False:
````
Because in the if syntax by default the if statement works  if the condition is true. but here we wanted the statement to work if the condition is false.   
I am a bit confused.